### PR TITLE
use babel-types v7

### PR DESCRIPTION
**Pull request by hzoo**
Fixes https://github.com/babel/babel/issues/6262

### CODE REVIEW

**Comment by hzoo** [comment]
cc @benjamn for https://github.com/babel/babel/issues/6262

**Comment by damassi** [comment]
Any update on merging this?

**Comment by hzoo** [comment]
This needs an update to peerDep on @babel/core and then just use that version instead of babel-types under a beta version.

@benjamn not sure what to do here because this also is causing bugs on our side because of the different versions of babel-types people are getting (also a bigger bundle in babel-standalone).

File: packages/regenerator-transform/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     ]
   },
   "dependencies": {
-    "babel-types": "^6.19.0",
+    "babel-types": "7.0.0-beta.2",
```
Comment (benjamn) [inline packages/regenerator-transform/package.json]: This should be `-beta.3` now, right?

**Comment by damassi** [comment]
Thanks @benjamn 👍

**Comment by hzoo** [comment]
Sorry this should of been `@babel/types` `v7.0.0-beta.32` now 😛

**Pull request merged**